app/controllers/course/levels_controller.rb

```diff
@@ -34,7 +34,7 @@ def destroy #:nodoc:
   # This methods ensures that the Course::Levels are numbered
   # for use by the controller.
   def load_levels
-    @levels = @course.numbered_levels
+    @levels = @course.levels
```

Comment:
Just to make sure I'm not misunderstanding things I could just place `@levels = @course.levels` straight in the index method? Since this is implied within the Levels model already.